component/src/main/java/com/synopsys/integration/alert/component/settings/convert/ProxyConfigurationModelSaveActions.java

```diff
@@ -0,0 +1,55 @@
+package com.synopsys.integration.alert.component.settings.convert;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.action.ActionResponse;
+import com.synopsys.integration.alert.common.action.api.GlobalConfigurationModelToConcreteSaveActions;
+import com.synopsys.integration.alert.common.persistence.model.ConfigurationModel;
+import com.synopsys.integration.alert.common.rest.AlertRestConstants;
+import com.synopsys.integration.alert.common.rest.model.SettingsProxyModel;
+import com.synopsys.integration.alert.component.settings.descriptor.SettingsDescriptorKey;
+import com.synopsys.integration.alert.component.settings.proxy.action.SettingsProxyCrudActions;
+import com.synopsys.integration.alert.descriptor.api.model.DescriptorKey;
+
+@Component
+public class ProxyConfigurationModelSaveActions implements GlobalConfigurationModelToConcreteSaveActions {
+    private final ProxyConfigurationModelConverter proxyFieldModelConverter;
+    private final SettingsProxyCrudActions configurationActions;
+
+    public ProxyConfigurationModelSaveActions(SettingsProxyCrudActions configurationActions) {
+        this.proxyFieldModelConverter = new ProxyConfigurationModelConverter();
```

Comment:
Make ProxyConfigurationModelConverter use the Component annotation and Autowire it into the construction.  You're also missing the Autowire annotation on the constructor